src/InvariantTest.sol

```diff
@@ -15,8 +14,21 @@ contract InvariantTest is Test {
     address[] private _targetedContracts;
     address[] private _targetedSenders;
 
+    string[] private _excludedArtifacts;
+    string[] private _targetedArtifacts;
+
+    FuzzSelector[] internal _targetedArtifactSelectors;
     FuzzSelector[] internal _targetedSelectors;
 
```

Comment:
Some of these vars are `private` and others are `internal`. Is there a reason for this difference? Wondering which we should prefer

Reply:
Good call out, I don't really see any advantage to `private`, what do you think? Who know someone might want access to these values in the inherited contracts.

Reply:
Hmm, I think I lean towards keeping it `private` until someone has a use case for needing access to the array directly, just a bit safer / more clear API that way since you can't accidentally overwrite it / directly set it